datafusion/functions-window/src/nth_value.rs

```diff
@@ -40,39 +40,28 @@ use std::hash::Hash;
 use std::ops::Range;
 use std::sync::{Arc, LazyLock};
 
-get_or_init_udwf!(
+define_udwf_and_expr!(
     First,
     first_value,
-    "returns the first value in the window frame",
+    [arg],
+    "Returns the first value in the window frame",
     NthValue::first
 );
-get_or_init_udwf!(
+define_udwf_and_expr!(
```

Comment:
Taking advantage of `define_udwf_and_expr!()` macro which creates the function for us; no change to public API, function definition is the same